indico/modules/events/abstracts/controllers/management.py

```diff
@@ -279,3 +281,29 @@ def _process(self):
         sorted_abstracts = sorted(self.abstracts, key=attrgetter('friendly_id'))
         pdf = ConfManagerAbstractsToPDF(self.event_new, sorted_abstracts)
         return send_file('abstracts.pdf', pdf.generate(), 'application/pdf')
+
+
+class RHAbstractsExportBase(RHManageAbstractsActionsBase):
+    """RH for all abstract list export classes"""
+
+    def _process(self):
+        self.export_config = self.list_generator.get_list_export_config()
+        self.headers, self.rows = generate_spreadsheet_from_abstracts(self.abstracts,
+                                                                      self.export_config['static_item_ids'],
+                                                                      self.export_config['dynamic_items'])
+
+
+class RHAbstractsExportCSV(RHAbstractsExportBase):
+    """Export list of abstracts to CSV"""
+
+    def _process(self):
+        RHAbstractsExportBase._process(self)
```

Comment:
Wouldn't it be cleaner to have a differently named methods for this which returns those headers and rows?

`return send_csv('abstracts.csv', *self._generate_spreadsheet())`